yoga/YGValue.h

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
Pulling this out into a separate file seems to not be needed for this PR. Could you revert this change?